webapp/portlet/src/main/webapp/vue-apps/popover/components/SpacePopoverContent.vue

```diff
@@ -55,6 +55,7 @@
         :is-favorite="space.isFavorite"
         :space-id="space.id" />
       <extension-registry-components
+        v-if="space.isMember"
```

Comment:
This will disable all extension components for non members, I prefer letting it to the logic of the extension component to display or not the extension, WDYT ?